src/lib/apiTypesV2/Subscription.cpp

```diff
@@ -97,6 +97,11 @@ std::string Subscription::toJson(void)
     jh.addNumber("throttling", this->throttling);
```

Comment:
Note that apart from modifications to toJson() to render the new fields, you would also need to modify the parsing logic for the new fields. Have a look to src/lib/jsonParseV2/parseSubscription.cpp

And of course documenting the new feature :), but let's wait for the final version of the PR to do that.

Reply:
Added parsing logic for the new field in https://github.com/telefonicaid/fiware-orion/pull/3890/commits/49b0c6df2c4d993b4e0fec160026fe0ac0017cb7